docs/releases/breaking-changes.md

```diff
@@ -13,6 +13,51 @@ these to a minimum, but rest assured we only make them with good reason.
 
 Starting with Directus 10.0, here is a list of potential breaking changes with remedial action you may need to take.
 
+## Version 10.10.0
+
+### Deprecated Local Extension Folders
```

Comment:
Depending on the outcome of https://github.com/orgs/directus/projects/26/views/1?pane=issue&itemId=52432308 we might need to expand a little